packages/cli/src/__tests__/checkpoint.test.ts

```diff
@@ -0,0 +1,42 @@
+import { jestConsoleContext, jestContext } from '@prisma/sdk'
+
+import { redactCommandArray, tryToReadDataFromSchema } from '../utils/checkpoint'
+
+const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+
+it('should redact --url [value]', () => {
+  expect(redactCommandArray(['cmd', '--url', 'redactme'])).toMatchInlineSnapshot(`
+    Array [
+      cmd,
+      --url,
+      [redacted],
+    ]
+  `)
+})
+
+it('should redact --url=[value]', () => {
+  expect(redactCommandArray(['cmd', '--url=redactme'])).toMatchInlineSnapshot(`
+    Array [
+      cmd,
+      --url=[redacted],
+    ]
+  `)
+})
```

Comment:
Maybe we should add a few more params here?

Reply:
Updated the test to test all options